### DISCUSSION TITLE
What dagster processes connect to the sql database?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#18279] [Q&A]
We are seeing sudden spikes in the number of connections to our dagster sql database, which dagster processes will be connecting to the database, is it just the daemons?

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1701096517329619?thread_ts=1701096517.329619&cid=C01U954MEER).*

**Comment by smackesey** [comment]
Both host and code-location processes connect. When a run is executed it will happen in a code location process, and events from the run are written to the database.

I can't provide any insight into the "sudden spikes" without more information.
